bot/internal/review/review.go

```diff
@@ -367,7 +390,7 @@ func (r *Assignments) CheckInternal(e *env.Environment, reviews []github.Review,
 		}
 	case changes.Docs && !changes.Code:
 		log.Printf("Check: Found docs changes.")
-		if err := r.checkInternalDocsReviews(e.Author, reviews); err != nil {
+		if err := r.checkInternalDocsReviews(e, reviews, files); err != nil {
```

Comment:
It's been a while since I've looked at this code and I'm pretty jet-lagged so forgive me if I'm missing something obvious - but why do we need to pass the set of files in for docs reviewers here but not for code reviewers?

Reply:
`checkInternalCodeReviews` gets all code reviewers for the team associated with the PR repo, then checks the reviewers who approved the PR for whether there are multiple code owners or one code owner and one reviewer within that team. It doesn't need the PR files, just the team assignments and reviews.

In `checkInternalDocsReviews, whether a reviewer counts as "internal" or not depends on the file paths within the changeset. Code reviewers are normally excluded from docs reviews unless they have `preferredReviewerFor` configured.